Template/function_definitions.inc

```diff
@@ -21,11 +21,10 @@ std::map< std::pair<int, int>, double > %(process_class)s::compute( const std::p
 
     // Suppose final particles are passed in the "correct" order
     std::vector<int> selectedFinalState(%(nexternal)s - 2);
-    size_t index = 2;
-    for (auto const &finalPart: finalState) {
-      selectedFinalState[index - 2] = finalPart.first;
-      momenta[index] = (double*) (&finalPart.second[0]);
-      index++;
+    for (size_t index = 0; index < (%(nexternal)s - 2); index++) {  
```

Comment:
You were a bit too prompt ;) I made some mistakes in my code snippet. The correct one is:

```
for (size_t index = 0; index < (%(nexternal)s - 2); index++) {
  selectedFinalState[index] = finalState[index].first; 
  momenta[index + 2] = (double * ) (&finalState[index].second[0]); 
}
```

(note: there's no `index++` no more)